src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotValues = Object.values(robot);
+
+  for (let i = 0; i < robotValues.length; i++) {
+    if (robotValues.indexOf(robotValues[i])
+      !== robotValues.lastIndexOf(robotValues[i])) {
+      return null;
+    }
+  }
```

Comment:
but i don't get the purpose of this piece of code actually) remove it

Reply:
i don`t know what i must to do... have any ideas? In my mind it`s most obvious solution.